benefits/static/css/styles.css

```diff
@@ -592,55 +681,79 @@ h1 + .media-list, /* A .media-list immediately following the h1: Enrollment Succ
   margin-left: var(--logout-paragraph-margin-left);
 }
 
-/* Agency Index */
-/* This page has a full-width and full-height background image, creating the need to re-set the Footer margin top to 0 */
+/* Index and Agency Index (Landing pages) */
+/* These pages have a full-width and full-height background image, creating the need to re-set the Footer margin top to 0 */
 
 :root {
-  --agency-index-main-height: 100vh;
-  --agency-index-background: url("/static/img/benefits-bg-mobile.jpg") no-repeat
+  --landing-main-height: 100vh;
+  --landing-background: url("/static/img/benefits-bg-mobile.jpg") no-repeat
     var(--primary-color);
+  --landing-box-background: var(--bs-white);
+  --landing-box-border: 12px solid var(--primary-color);
+  --landing-text-color: var(--text-primary-color);
   --agency-index-box-background: var(--bs-white);
   --agency-index-box-border: 12px solid var(--primary-color);
   --agency-index-text-color: var(--text-primary-color);
+  --landing-page-x-margin: auto;
 }
 
 @media (min-width: 992px) {
   :root {
-    --agency-index-main-height: calc(
+    --landing-main-height: calc(
       100vh - 130px
     ); /* 130px = Header Height (80px) + (Footer Link (50px) */
-    --agency-index-background: url("/static/img/benefits-bg-desktop.jpg")
-      no-repeat;
-    --agency-index-box-background: transparent;
-    --agency-index-box-border: none;
-    --agency-index-text-color: var(--bs-white);
+    --landing-background: url("/static/img/benefits-bg-desktop.jpg") no-repeat;
+    --landing-box-background: transparent;
+    --landing-box-border: none;
+    --landing-text-color: var(--bs-white);
+    --landing-page-x-margin: 0;
   }
 }
 
-.agency-index main#main-content {
-  min-height: var(--agency-index-main-height);
-  background: var(--agency-index-background);
+.landing main#main-content {
+  min-height: var(--landing-main-height);
+  background: var(--landing-background);
   background-size: cover;
 }
 
-.agency-index main#main-content .container .row:first-child {
-  min-height: var(--agency-index-main-height);
+.landing main#main-content .container .row:first-child {
+  min-height: var(--landing-main-height);
 }
 
-.agency-index .box {
-  background: var(--agency-index-box-background);
-  border-top: var(--agency-index-box-border);
-  color: var(--agency-index-text-color);
+.landing .box {
+  background: var(--landing-box-background);
+  border-top: var(--landing-box-border);
+  color: var(--landing-text-color);
   margin-bottom: calc(32rem / 16);
 }
 
-.agency-index .box .btn.btn-lg.btn-primary {
+.landing .box .btn.btn-lg.btn-primary {
   width: 250px; /* Button is 250 x 72 */
   display: block;
-  margin: 0 auto; /* Center the button */
+  margin: 0 var(--landing-page-x-margin);
   padding: 19px 0; /* Button is 250 x 72 */
 }
 
-.agency-index footer {
+.landing footer {
   margin-top: 0;
 }
+
+/* Agency Selector */
+
+#agency-selector .modal-body {
+  padding: 0px;
```

Comment:
No `px` needed

Reply:
Done in 691f66d1427c85d1cc1b7e4bca48beb10f6a9621